exercises/largest-series-product/example.rs

```diff
@@ -0,0 +1,24 @@
+use std::str::FromStr;
+
+pub fn lsp(string_digits: &str, span: usize) -> Result<u64, String> {
+    if span == 0 {
+        return Ok(1);
+    }
+
+    if string_digits.chars().any(|c| !c.is_numeric()) {
```

Comment:
`char::is_numeric` accepts all kinds of Unicode characters (see [doc](https://doc.rust-lang.org/std/primitive.char.html#method.is_numeric)). I propose using `c.is_digit(10)` here.